mgmtsystem/models/res_config.py

```diff
@@ -107,3 +107,20 @@ class MgmtsystemConfigSettings(models.TransientModel):
         help="Provide Work Instructions category.\n"
         "- This installs the module document_page_work_instruction.",
     )
+
+    def execute(self):
+        # Provide error in case the odule to install is not available in the system
+        # This avoids user confusion from the install failing silently
+        self = self.with_context(active_test=False)
+        classified = self._get_classified_fields()
+        to_install = [
+            f[7:] for f in self._fields.keys() if f.startswith("module_") and self[f]
+        ]
+        available = classified["module"].mapped("name")
+        not_available = set(to_install) - set(available)
+        if not_available:
+            raise exceptions.UserError(
+                _("The following modules are not available: %s")
```

Comment:
@dreispt Can we add a message pointing to the Migration issue on Github and asking people to leave a comment if they want to contribute?